chore: Rebrand to Wynntils (move away from using the Artemis codename) (#2711)
Wynntils/Wynntils: README.md

Wynntils
========
[![Discord](https://discordapp.com/api/guilds/394189072635133952/widget.png)](https://discord.gg/ve49m9J)
[![CurseForge](https://cf.way2muchnoise.eu/short_wynntils.svg)](https://www.curseforge.com/minecraft/mc-mods/wynntils)
[![Modrinth](https://img.shields.io/modrinth/dt/Wynntils?label=modrinth)](https://modrinth.com/mod/wynntils)
[![GitHub](https://img.shields.io/github/downloads/Wynntils/Artemis/total?logo=github)](https://github.com/Wynntils/Artemis/releases)
[![License](https://img.shields.io/badge/license-LGPL%203.0-green.svg)](https://github.com/Wynntils/Wynntils/blob/main/LICENSE)

<div align="center">

> <img src="https://upload.wikimedia.org/wikipedia/commons/d/d2/Artemis.png" width=20%>

Reviewer: ```suggestion
```

Wynntils
========
[![Discord](https://discordapp.com/api/guilds/394189072635133952/widget.png)](https://discord.gg/ve49m9J)
[![CurseForge](https://cf.way2muchnoise.eu/short_wynntils.svg)](https://www.curseforge.com/minecraft/mc-mods/wynntils)
[![Modrinth](https://img.shields.io/modrinth/dt/Wynntils?label=modrinth)](https://modrinth.com/mod/wynntils)
[![GitHub](https://img.shields.io/github/downloads/Wynntils/Artemis/total?logo=github)](https://github.com/Wynntils/Artemis/releases)
[![License](https://img.shields.io/badge/license-LGPL%203.0-green.svg)](https://github.com/Wynntils/Wynntils/blob/main/LICENSE)

<div align="center">

<img src="http://cdn.wynntils.com/wynntils%20logo%20-%20w%20paintstroke.png" width=15% alt="Wynntils Logo">

> 

Reviewer: We need to move the `</div>` to right after the img, otherwise part of the text gets centered as well. 

(Or maybe that was your intention? It looks odd, imho, though)

Author: It was kind of the intention. Sure, Im okay with it either way.

Reviewer: Thanks. We can go over the README separately afterwards, and see if we can highlight this paragraph in another way that does not look too odd.

> > Artemis is the greek goddess of hunting and the moon, born of Zeus and Leto, twin of Apollo.

Reviewer: ```suggestion
```

Reviewer: Actually, I think we should start with some paragraph stating the overall purpose of Wynntils. Something like:

Wynntils is a Minecraft mod for the MMORP [Wynncraft](https:///wynncraft.com). It is intended to enhance your gaming experience when playing Wynncraft.

Reviewer: Or maybe reuse the wording from legacy: "Wynntils is a Wynncraft mod that seeks to enhance the user's gameplay with a variety of customizable options and additions."

Maybe we can also replace the Artemis image with this from the Legacy readme:

![logo](https://camo.githubusercontent.com/b7f2c73e3dadf738c755e8bb01bf2c5460be9641a4612f1f0b98dd85db352c34/687474703a2f2f63646e2e77796e6e74696c732e636f6d2f77796e6e74696c732532306c6f676f2532302d253230772532307061696e747374726f6b652e706e67)

Reviewer: ... perhaps after scaling it down a bit :)

> Wynntils is a rewrite of **[Wynntils](https://github.com/Wynntils/Wynntils)** (informally referred to as "Legacy") in 1.21 using Architectury, to support **Fabric** and **NeoForge**. Artemis was a codename used for the rewrite of Wynntils.

Reviewer: ```suggestion
Wynntils is based on Minecraft 1.21 using Architectury, and supports **Fabric** and **NeoForge**. This repository contains a rewrite (originally codenamed Artemis) of the original Wynntils mod, now known as [Wynntils Legacy](https://github.com/Wynntils/Wynntils-Legacy) .
```
